maraboupy/Marabou.py

```diff
@@ -121,10 +117,22 @@ def solve_query(ipq, filename="", verbose=True, options=None):
 
     return [vals, stats]
 
-def createOptions( numWorkers=4, initialTimeout=5, initialDivides=0, onlineDivides=2,
+def createOptions(numWorkers=4, initialTimeout=5, initialDivides=0, onlineDivides=2,
                    timeoutInSeconds=0, timeoutFactor=1.5, verbosity=2, dnc=False):
-    """
-    Create an option object
+    """Create an options object for how Marabou should solve the query
+
+    Args:
+        numWorkers (int, optional): Number of workers to use in DNC mode, defaults to 4
+        initialTimeout (int, optional): Initial timeout in seconds for DNC mode before dividing, defaults to 5 
+        initialDivides (int, optional): Initial divides in DNC mode, defaults to 0
+        onlineDivides (int, optional): Online divides in DNC mode, defaults to 2
+        timeoutInSeconds (int, optional): Timeout duration for Marabouin seconds, defaults to 0
+        timeoutFactor (float, optional): Timeout factor for DNC mode, defaults to 1.5
+        verbosity (int, optional): Verbosity level for Marabou, defaults to 2
+        dnc (bool, optional): If DNC mode should be used, defaults to False
+
```

Comment:
Could we change the description of initialDivides to "Number of times to perform the initial partitioning. This creates 2^(initialDivides) sub-problems, defaults to 0" 
onlineDivides to "Number of times to perform the online partitioning when a sub-query times out. This creates 2^(onlineDivides) sub-problems, defaults to 2" .

Just to highlight that this is how many times we bisect the problem, not the number of created sub-problems.

Reply:
Changes made, thanks for the suggestion.